app/src/components/createLearningJourney/index.jsx

```diff
@@ -25,12 +25,12 @@ export default function index() {
   return (
     <div className='flex flex-col container w-9/12 max-w-7xl mt-10 p-10 mx-auto w-full bg-white rounded-lg shadow-lg shadow-blue-200 justify-around'>
       <h1 className='text-3xl text-left font-bold'>Create your Learning Journey</h1>
-      <CreateLJBody selectedJobRole={selectedJobRole} />
+      <CreateLearningJourney selectedJobRole={selectedJobRole} />
```

Comment:
Renaming this to `CreateLearningJourney` may be confusing since the `index()` would be what is actually referenced as the `CreateLearningJourney` from the routes

Reply:
Reverted